src/mlpack/tests/mlpack_test.cpp

```diff
@@ -47,6 +106,22 @@ struct GlobalFixture
       mlpack::Log::Info.ignoreInput = true;
       mlpack::Log::Warn.ignoreInput = true;
     #endif
+
+    for (int i = 0; i < boost::unit_test::framework::master_test_suite().argc;
+        i++)
+    {
+      std::string argument(
+          boost::unit_test::framework::master_test_suite().argv[i]);
+
+      // Print Boost test hierarchy.
+      if (argument == "tests")
```

Comment:
Do you know an easy way to register a new command line argument for ``print_tests``?